pkg/reconciler/messaging/channel/channel.go

```diff
@@ -326,10 +376,6 @@ func (r *Reconciler) getTopic(_ context.Context, channel *v1beta1.Channel) (*int
 	if err != nil {
 		return nil, err
 	}
-	if !metav1.IsControlledBy(topic, channel) {
```

Comment:
Why is this being removed?

Reply:
Moved it to the caller code to better match other reconcilers. Do you prefer it here?

Reply:
This is fine, thanks for explaining.